ocean_lib/ocean/ocean_exchange.py

```diff
@@ -132,7 +136,9 @@ def setRate(
         # get tx receipt
         tx_receipt = exchange.get_tx_receipt(self._web3, tx_id)
         # get event log from receipt
-        logs = exchange.contract.events.ExchangeRateChanged().processReceipt(tx_receipt)
+        logs = exchange.contract.events.ExchangeCreated().processReceipt(
```

Comment:
Please change this back to `ExchangeRateChanged()`.  This is why `test_ocean_exchange` is failing.

Reply:
Missed that, ok.